samples/Sentry.Samples.Console.Basic/Sentry.Samples.Console.Basic.csproj

```diff
@@ -35,5 +35,8 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\Sentry\Sentry.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```

Comment:
Is this needed for modern .NET or just .NET Framework?

Reply:
You're right... I can add `Condition="'$(TargetFramework)' == 'net462'"` here.